tests/DependencyLoaderTest.php

```diff
@@ -6,28 +6,55 @@
 
 use Dominikb\ComposerLicenseChecker\Contracts\DependencyParser;
 use Dominikb\ComposerLicenseChecker\DependencyLoader;
+use Dominikb\ComposerLicenseChecker\Exceptions\CommandExecutionException;
 use Mockery;
+use Symfony\Component\Console\Command\Command;
 
 class DependencyLoaderTest extends TestCase
 {
-    /** @test */
+    /**
+     * @test
+     *
+     * @requires OS Linux
```

Comment:
```suggestion
     * @requires OS Linux|Darwin
```

Reply:
Will switch to `OSFAMILY Linux|Darwin`, even though I haven't validated the same shell escaping applies.